tests/functional/restart/58-removed-task.t

```diff
@@ -39,10 +39,9 @@ workflow_run_ok "${TEST_NAME}" cylc play --no-detach "${WORKFLOW_NAME}"
 TEST_NAME="${TEST_NAME_BASE}-restart"
 workflow_run_ok "${TEST_NAME}" cylc play --set="INCL_B_C=False" --no-detach "${WORKFLOW_NAME}"
 
-grep_workflow_log_ok "grep-3" "\[1/a running job:01 flows:1\] (polled)started"
-grep_workflow_log_ok "grep-4" "\[1/b failed job:01 flows:1\] (polled)failed"
+grep_workflow_log_ok "grep-3" "=> failed"
```

Comment:
Having read what the test is trying to do I think this would be more explicit
```suggestion
grep_ok '\[jobs-poll out\] .* "run_status": 0'  "${WORKFLOW_RUN_DIR}/log/job/1/a/NN/job-activity.log"
grep_ok '\[jobs-poll out\] .* "run_status": 1'  "${WORKFLOW_RUN_DIR}/log/job/1/b/NN/job-activity.log"
grep_workflow_log_ok "grep-3" "\[1/b .* => failed"
```